middle_end/flambda/inlining/inlining_decision.ml

```diff
@@ -33,9 +55,9 @@ module DE = Downwards_env
 module Function_declaration_decision = struct
   type t =
     | Never_inline_attribute
-    | Function_body_too_large of Inlining_cost.Threshold.t
+    | Function_body_too_large of Code_size.t
     | Stub
-    | Inline of (int * Inlining_cost.Threshold.t) option
+    | Inline of (Code_size.t * Code_size.t) option
```

Comment:
This should be renamed to `Can_inline` since we now perform a non-trivial decision at call site.

Reply:
Done. I also added more fields to describe a bit more the different decisions.